styleguide/source/_patterns/03-templates/04-topic/topic.twig

```diff
@@ -9,22 +9,25 @@
     </div>
   </div>
 
-  <div class="topic_main grid grid-margin">
```

Comment:
I'm not a huge fan of "grid" classes. What are your thoughts on @extend "grid and grid-margin" inside of "topic_main" class and others? Thoughts?

Reply:
I think we should rethink the class names in SG2. I tried to as little to disrupt the patterns that are already in place in SG.

Reply:
That is the correct approach. However any notes that you would like to see implemented in SG2 please add notes to https://issues.ama-assn.org/browse/EWL-4103

Reply:
> @extend "grid and grid-margin" inside of "topic_main"

IIRC we chose to use named classes like this so it'd be apparent where grids get instantiated from a glance at the markup. And then the `.topic_main` type classes were added as a last resort if something couldn't be selected using another selector. Just supplying a little context to aid in future discussion maybe.